src/subsplit_dag.cpp

```diff
@@ -120,28 +120,43 @@ std::string SubsplitDAG::ToDot(bool show_index_labels) const {
   return string_stream.str();
 }
 
-const BitsetSizeMap &SubsplitDAG::GetGPCSPIndexer() const { return gpcsp_indexer_; }
-const BitsetSizePairMap &SubsplitDAG::GetParentToRange() const {
-  return parent_to_range_;
+BitsetSizeMap SubsplitDAG::GetGPCSPIndexer() const {
```

Comment:
Could we rename this `BuildGPCSPIndexer` to make it clear that there is some nontrivial computation happening?